puppet/puppet.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# Created by Wesley Fisher. praetor@sdf.org @gwesfisher
+
+apt-get update
+
+apt-get install -y puppet libssl-dev
+
+if [ -f /etc/puppet/puppet.conf ]; then
+	echo "[agent]" >> /etc/puppet/puppet.conf
+	echo "server=$1" >> /etc/puppet/puppet.conf
+fi
+
+puppet agent --enable
+
+if [ $2 -eq "Yes"]; then
```

Comment:
This line throws an error, so I'm guessing there's supposed to be a space between the `"Yes"` and `]`?

I tried adding a space in and then got the error `puppet.sh: 16: [: Illegal number: Yes` so I'm wondering if there should be quotes round the `$2` as well?